AvatarGenerator/po/it-local.po

```diff
@@ -0,0 +1,161 @@
+# Addon AvatarGenerator for Gramps.
+# Copyright (C) 2025 Gramps
+# This file is distributed under the same license as the gramps package.
+#
+#
+# Sebastiano Pistore <SebastianoPistore.info@protonmail.ch>, 2025.
+msgid ""
+msgstr ""
+"Project-Id-Version: it\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2019-06-14 11:49+1000\n"
+"PO-Revision-Date: 2025-01-04 17:35+0100\n"
+"Last-Translator: Sebastiano Pistore <SebastianoPistore.info@protonmail.ch>\n"
+"Language-Team: \n"
+"Language: it_IT\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Poedit-SourceCharset: utf-8\n"
+"X-Generator: Poedit 3.5\n"
+
+#: AvatarGenerator/AvatarGenerator.gpr.py:23
+msgid "Avatar Generator Tool"
+msgstr "Avatar Generator Tool"
```

Comment:
Please translate this one as well, the name attribute is translated for the buildin gramplets

Reply:
> Please translate this one as well, the name attribute is translated for the buildin gramplets

I thought about leaving it in English because it is the name of the program, but in fact the names of other addons are translated. It sounds a little strange to me.